src/util/HttpServer/HttpUtils.h

```diff
@@ -77,15 +77,17 @@ static auto createOkResponse(std::string text, const HttpRequest auto& request,
                                       request, mediaType);
 }
 
+/// Assigns the generator to the body, potentially applying compression
+/// along the way and setting the appropriate headers
```

Comment:
...the body -> of the `response`. If a supported compression method is specified in the `request`, this method is applied to the `body` and the corresponding `response` headers are set.